File: src/main/java/com/wxl/dyttcrawler/domain/DyttReleaseDate.java
package com.wxl.dyttcrawler.domain;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.Date;

/**
 * Create by wuxingle on 2020/5/3
 * 上映时间
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class DyttReleaseDate implements Serializable {

    private static final long serialVersionUID = -5634712742357231640L;

    // 时间
    @JsonFormat(pattern = "yyyy-MM-dd")
    private Date date;

    // 地区
    private String place;
}

File: src/main/kotlin/com/wxl/dyttcrawler/domain/DyttReleaseDate.kt
package com.wxl.dyttcrawler.domain

import com.fasterxml.jackson.annotation.JsonFormat
import java.util.*

/**
 * Create by wuxingle on 2021/10/12
 * 上映时间
 */
data class DyttReleaseDate(

    // 时间
    @JsonFormat(pattern = "yyyy-MM-dd")
    var date: Date? = null,

    // 地区
    var place: String? = null
)
